roles/tuned/tasks/main.yml

```diff
@@ -13,7 +13,7 @@
     set_fact:
       openshift_tuned_guest_profile: "{{ 'atomic-guest' if openshift.common.is_atomic else 'virtual-guest' }}"
       tuned_etc_directory: '/etc/tuned'
-      tuned_templates_source: '../templates/tuned'
+      tuned_templates_source: '../templates'
```

Comment:
This variable is probably unneeded.  In any case, it should be in role defaults, not set_fact.